packages/vue/src/components/SpCrud/SpCrud.vue

```diff
@@ -117,11 +118,15 @@ export default defineComponent({
 
     // state
     let state: State = reactive(initialState)
+    let moduleName = computed (() =>
```

Comment:
Let's either use `item` or `module` for naming this. 

For the case insensitive version, let's use something like `<name>Normalized `

Reply:
also mind placing the computed in a separate code block following ` // computed` comment